### DISCUSSION TITLE
Display by Hovering has stopped working in Notes and Details

### DISCUSSION THREAD

**Discussion by arresse** [freeplane/freeplane#1085] [Help]
I have made extensive use of Hovering over Note and Details nodes but for some reason this feature has stopped working. Notes and Details can still be displayed otherwise, but will not appear when hovering.

I'm wondering if this may have something to do with the size of my map which is approaching 12 MB. I've tried starting a new map but even there the hover feature does not work.

**Comment by EdoFro** [comment]
look at the tool tips preferences in
`Preferences…->Behaviour->Tooltips->Display tool tips`

I have this configuration and it works in my computer:
![image](https://user-images.githubusercontent.com/35700575/231294267-70b8abed-6e2a-4b20-827f-a9529498f632.png)

**Comment by sifran-github** [comment]
Check the options under:
view->tooltips-> ...

**Comment by arresse** [comment]
Yes that's it, thank you. Strange, because I have never disabled these functions.
